WORKSPACE

```diff
@@ -2062,6 +2015,28 @@ rpm(
     ],
 )
 
+rpm(
+    name = "kde-filesystem-0__4-63.fc32.aarch64",
```

Comment:
That is a funny package :D

Reply:
I thought so as well! Got pulled in as a dependency, I guess.

Reply:
Just for my own curiosity

https://src.fedoraproject.org/rpms/libisoburn/blob/rawhide/f/libisoburn.spec#_64